### ISSUE TITLE
Niggly Powershell issues

### ISSUE THREAD

**Issue by talentoscope** [RainbowMiner/RainbowMiner#290]
![image](https://user-images.githubusercontent.com/22162766/48149617-77ff2280-e2b4-11e8-8d17-b9e1efb4ed01.png)

Question: What does this mean at the top with RainbowMiner and an amount, and then when my card is active, the miner sits underneath? What is being compared? Also, sometimes a yellow line pops up and says "This is x% more profitable than Y". I don't get what that means, and it doesn't seem to be explained anywhere. Hope I'm not being dense :)

Issue: The colour code under Sym aren't working, though they work elsewhere. There's a mixture of BTC which are autoexchange coins and mined coins there, if that makes any difference. Great that these show up though - would be awesome if that could be shown in the pools on the web dashboard by coin.

Okay, that's 3 issues. Sorry! :cry: :)

**Comment by talentoscope** [comment]
Also the bit above with the last ran miners gets quite large sometimes, especially after benchmarking. Would be great if it could be kept to a maximum of 1, or clear after 5 mins or something so we can see the profit calcs at the very top.

No other problems, I swear ;)

**Comment by RainbowMiner** [comment]
> Regarding these strange color symbols and the number underneath: which version of powershell and windows are you using?

.. in addition: please goto http://localhost:4000 and click "Debug file" at the left bar and drag-n-drop the created 7z file here. It will contain all current logs plus the running config with all critical values x-ed out, so no api keys, nor wallets will be exposed.

**Comment by talentoscope** [comment]
Windows 7 x64 and the version of Powershell that comes with Windows by default. It worked so didn't want to fiddle with anything
Debug attached :)
[debug-2018-11-08.zip](https://github.com/RainbowMiner/RainbowMiner/files/2562645/debug-2018-11-08.zip)
Had to recompress to zip, GitHub doesn't like 7zip

**Comment by talentoscope** [comment]
Okay v6.1 and it's the same, but the other colours are fine. Not something to do with the dashes in the table or something silly is it? I mean it did work before those were shown when everything was BTC iirc.

**Comment by RainbowMiner** [comment]
Thank you :) Sorry for 7z archives. I had forgotten that github likes zip only. I already changed the zipping for the next release.

It is this:
> Windows 7 x64

.. the color syntax codes have been added with Windows 10 to the console. For Windows 7 you would need a cheat. I think it is better, I detect if Windows 7 is being used and then disable the color coding (will do this for the next release).
